content/concepts/asset-pricing.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Asset pricing models
+description: Choose one of the pricing models during asset publishing
+---
+
+Ocean Protocol offers 3 types of pricing options for asset monetization. The asset publisher can choose one of the available options which best suits their needs while publishing an asset. Once the pricing type is set, and the asset is published, the pricing model is permanently fixed for the asset and cannot be changed later.
+
+## Fixed pricing
+
+In a fixed pricing model, an asset's value is fixed as long as it is available in the market for buying. Publisher can set the price by deciding the value of a datatoken in terms of the number of Ocean tokens. The buyer needs to pay the exact number of Ocean tokens to get 1 datatoken. The publisher has an option to update the price of the dataset later anytime.
+
+Publishers can choose this fixed pricing model when they do not want Automated Market Maker(AMM) pools to decide the price discovery. If the publisher has already analyzed and estimated the worth of the dataset and is ready to sell an asset at a constant price, this is the suitable pricing model.
+
+The below image shows how to set the fixed pricing of an asset in the Ocean's Marketplace. Here, the price of the asset is set to 10 Ocean tokens.
+
+![fixed-asset-pricing](images/fixed-asset-pricing.png 'Fixed asset pricing using Marketplace')
+
+
+The publisher creates an Automated Market Maker(AMM) pool of Datatoken and Ocean tokens in dynamic pricing. *AMM* enables unstoppable, decentralized trading of assets in the liquidity pool.
```

Comment:
careful with the terms "assets" now, cause in this document we use "asset" first to describe "our" assets, published datasets or algorithms. So would try to avoid using "asset" in another context here describing the token composition of the liquidity pool.